docs/src/pages/demos/tooltip/tooltip.md

```diff
@@ -0,0 +1,11 @@
+---
+components: Tooltip
+---
+
+# Tooltip
```

Comment:
`Tooltips` so we follow the official documentation website, same goes with the folder and filename